TheMET/TheMET/DataModels/Department.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Department.swift
+//  TheMET
+//
+//  Created by Анна Ситникова on 21/03/2023.
+//
+
+import Foundation
+
+class Department: Decodable {
+    
+    let departmentId: Int
+    
+    let displayName: DisplayName
+    
+    let imageURL: URL?
+    
+    let numberOfObjects: Int
```

Comment:
As I see, there is no way to get the amount of objects from API

Reply:
Not sure, let's remove this option yet and discuss it later.